packages/server/src/analytics/logic.test.ts

```diff
@@ -0,0 +1,80 @@
+import { analyticsStorage } from './storage'
+import { mocked } from 'ts-jest/utils'
+import { analyticsLogic } from './logic'
+import { createMock } from 'ts-auto-mock'
+
+jest.mock('./storage')
+
+type Snapshot = FirebaseFirestore.QuerySnapshot<FirebaseFirestore.DocumentData>
+type SignupMock = { createTime: { seconds: number } }
+
+// Returns a snapshot that always has 40 yesterday signups and 500 total signups.
```

Comment:
Isn't it better to say "40 signups yesterday and 460 at lastQueryTime"?
This is a very strange function.  Can it just take 2 numbers for number of entries from yesterday and before yesterday?  That would make the interface a lot clearer.